Source/Particles/ParticleBoundaries.H

```diff
@@ -10,8 +10,34 @@
 #include "Utils/WarpXAlgorithmSelection.H"
 
 #include <AMReX_AmrCore.H>
+#include <AMReX_Array.H>
+#include <AMReX_Config.H>
 #include <AMReX_Parser.H>
 
+#include <utility>
+
+namespace warpx::particles
+{
+    /**
+     * \brief Reads the particle boundary conditions from
+     * the inputfile
```

Comment:
This can probably fit on one line as well?
```suggestion
     * \brief Reads the particle boundary conditions from the input file
```